src/ResourceManager/Compute/Commands.Compute/Generated/Snapshot/Config/SetAzureRmSnapshotUpdateImageReferenceCommand.cs

```diff
@@ -28,6 +28,8 @@
 
 namespace Microsoft.Azure.Commands.Compute.Automation
 {
+    [Obsolete("This cmdlet will be removed in the upcoming release because image reference cannot be updated for snapshot.")]
```

Comment:
Please explain thoroughly what is changing, why it is changing, and how the user would accomplish the task that they were trying to accomplish with this cmdlet.  For example, do they now need to create a new disk to get a disk with a different SnapshotUpdateImageReference?  Is there some other means of accomplishign the same task?  Do we have documentation we can link to about the change?